### ISSUE TITLE
Add folder icons to seti theme

### ISSUE THREAD

**Issue by AshleyScirra** [Microsoft/vscode#11767]
- VSCode Version: 1.5.1
- OS Version: Windows 10

Steps to Reproduce:

1. Use the new seti file icon theme
2. Keep making mistakes due to the fact files have icons, but not folders, which makes files appear indented. This makes files which are siblings to folders look like they're actually contained within the folders, because they are indented by their icon but the folders are not. So using this theme makes things super confusing in the explorer.
3. Switch to a different theme because of this issue despite liking the colorful style.

tl;dr please add folder icons to the seti theme :) you could just copy the ones from the "minimal" theme!

**Comment by AshleyScirra** [comment]
Oops, looks like a dupe of #11762.
